src/utility_belt/files.clj

```diff
@@ -0,0 +1,39 @@
+(ns utility-belt.files
```

Comment:
It's very rare that we will need to deal with files read from the file system, so we can reduce the scope of this ns and rename it to `resources`, and just provide simple utils powered by `io/resource` and `slurp` to read given data format, with error checking of course.